enterprise/models.py

```diff
@@ -1244,6 +1250,24 @@ def _complete_user_enrollment():
                         discount_percentage=enrollment.discount_percentage,
                         sales_force_id=enrollment.sales_force_id,
                     )
+
+                    # If the pending enrollment was using a subscription license, generate a license enrollment
+                    # alongside the enterprise enrollment.
+                    if enrollment.license_uuid:
+                        source = EnterpriseEnrollmentSource.get_source(EnterpriseEnrollmentSource.ENROLLMENT_URL)
```

Comment:
I see enrollment_source is alrady passed as a kwargs to this function when called elsewhere in the pr, do you think we need to re-fetch this here?

Reply:
Sorry, do you know where in this PR? I'm looking now.

Reply:
are you talking about how the enrollment from the pending enrollment has a source?

Reply:
yeah I was confused why we fetched this source here since it looks like source is being passed as a param too? https://github.com/edx/edx-enterprise/pull/1176/files#diff-e3af55e5dcd947b5e06b20381920b6c81bb4f189e2cceed0c6f2309c4051ff84R1487 I could be misreading this but worth a quick pass on correctness